src/Grid/DialogContent.js

```diff
@@ -0,0 +1,61 @@
+import Button from 'material-ui/Button';
+import DialogInput from './DialogInput.js';
+import React from 'react';
+import moment from 'moment';
+
+class DialogContent extends React.Component {
+  render(){
+    let value = '';
+    let value2 = '';
+    
+    if(this.props.columnType === 'datetime' || this.props.columnType === 'date' || this.props.columnType === 'datetimeutc'){
+      value = this.props.value === undefined ? moment().format() : this.props.value;
+      value2 = this.props.value2 === undefined ? moment().format() : this.props.value2;
+    }
+    else if(this.props.columnType === 'boolean'){
+      value = this.props.value === undefined ? '' : this.props.value;
+    }
+    else{
+      value = this.props.value === undefined ? '' : this.props.value;
+      value2 = this.props.value2 === undefined ? '' : this.props.value2;
+    }
+
+    return (
+      <div >
+        <DialogInput
+          value={value} 
+          mod={'Value'}
+          label={'Value'} 
+          classes={this.props.classes} 
+          columnType={this.props.columnType}
+          activeFilter={this.props.activeFilter} 
+          handleDatePicker={this.props.handleDatePicker}
+          handleBooleanDropDown={this.props.handleBooleanDropDown}
+          handleTextFieldChange={this.props.handleTextFieldChange} 
+        />
+        
+        {this.props.operator === 'Between' ? 
```

Comment:
Use short-circuit when the second path is null... For example:

`condition ? <Component /> : null;`

can be coded like:

`condition && <Component />;`